tests/JMS/Serializer/Tests/Metadata/Driver/yml/Discriminator.ObjectWithXmlAttributeDiscriminatorParent.yml

```diff
@@ -0,0 +1,8 @@
+JMS\Serializer\Tests\Fixtures\Discriminator\ObjectWithXmlAttributeDiscriminatorParent:
+    discriminator:
+        field_name: type
+        map:
+            child: JMS\Serializer\Tests\Fixtures\Discriminator\ObjectWithXmlAttributeDiscriminatorChild
+    xml_discriminator:
+        attribute: true
+        cdata: false
```

Comment:
My idea is just to keep it consistent. I would like to keep consistency within a format mainly. 
Having  consistency across formats is nice to have. 

Since XML and YAML formats have already differences,  and users use only one of them, will try to keep them consistent with the rest of the format itself.